.github/workflows/bloat.yaml

```diff
@@ -0,0 +1,75 @@
+name: Bloat Check
+on:
+  pull_request:
+
+jobs:
+  build:
+    name: Bloat Check
+    runs-on: ubuntu-latest
+    outputs:
+      base_build_dir: ${{ steps.build_master.outputs.build_dir }}
+      current_build_dir: ${{ steps.build_branch.outputs.build_dir }}
+
+    permissions:
+      contents: read
+
+    container:
+      image: ghcr.io/gravitational/teleport-buildbox:teleport14
```

Comment:
Depends. The size of the binary should be the same when you add a Go function to Teleport. Differences in size can also include:
* Go toolchain update - different versions may change the overall binary size.
* C libraries - different images can technically include different 3rd party dependencies like libbpf, fido etc. 

I think that our release image/build is closer to "reality" rather than our CI build. I don't think the differences are huge currently, but since we finally stopped using our devbox for everything this may change.